source/globalCommands.py

```diff
@@ -1184,6 +1184,29 @@ def script_review_endOfLine(self,gesture):
 	script_review_endOfLine.__doc__=_("Moves the review cursor to the last character of the line where it is situated in the current navigator object and speaks it")
 	script_review_endOfLine.category=SCRCAT_TEXTREVIEW
 
+	def script_review_currentSymbol(self,gesture):
+		info=api.getReviewPosition().copy()
+		info.expand(textInfos.UNIT_CHARACTER)
+		text = info.text
+		curLanguage = None
+		if config.conf['speech']['autoLanguageSwitching']:
+			for field in info.getTextWithFields({}):
+				if isinstance(field,textInfos.FieldCommand) and field.command=="formatChange":
+					curLanguage=field.field.get('language')
+		if curLanguage is None:
+			curLanguage = speech.getCurrentLanguage()
+		expandedSymbol = characterProcessing.processSpeechSymbol(curLanguage, text)
+		repeats=scriptHandler.getLastScriptRepeatCount()
+		if repeats == 0:
+			# Explicitly tether here
+			braille.handler.setTether(braille.handler.TETHER_REVIEW, auto=True)
+			speech.speakTextInfo(info,unit=textInfos.UNIT_CHARACTER,reason=controlTypes.REASON_CARET)
+		elif expandedSymbol != text:
+			# Translators: title for expanded symbol dialog.
+			ui.browseableMessage("%s\n%s" % (text, expandedSymbol), _("Expanded symbol %s" % languageHandler.getLanguageDescription(curLanguage)))

```

Comment:
For speech users, I think it is confusing that both the original and replacement are shown here. I initially thought that the replacement was just duplicated.

```suggestion
			ui.browseableMessage(_(
				"Character: {character}\n"
				"Replacement: {replacement}"
			).format(character=text, replacement=expandedSymbol),
			_("Expanded symbol ({language})").format(
				language=languageHandler.getLanguageDescription(curLanguage)
			))
```